docs/topics_5/web-add-applications.adoc

```diff
@@ -3,24 +3,38 @@
 [id='add_applications_{context}']
 = Adding applications to a project
 
-Specify the applications to add to this project. You can upload applications or register a server path that contains applications.
+A project must contain one or more applications. You cannot create a "dummy" project that contains no applications.
+
+You can add applications to a migration project by using either or both of the following methods:
+
+* Uploading applications
+* Registering the directory path of each application
+
+Uploading applications stores them directly in the {ProductShortName} server.
+
+Registering the directory path of an application tells the {ProductShortName} server the application's location. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
```

Comment:
'application's' - IBM says we're not supposed to use possessive apostrophe-s with inanimate objects. I think the rule is silly but we hav eto follow it.